### ISSUE TITLE
💡 [REQUEST] - <Search scene controller>

### ISSUE THREAD

**Issue by Bmcl755** [SE310-Team1/ShoppIt#25]
### Start Date

11/08/2022

### Summary

Create a search scene javaFX controller

### Basic Example

This will allow us to control the scene's transitions and buttons

**Comment by scho518** [comment]
Dependent on #21.

**Comment by scho518** [comment]
Superseded by #54.
